README.rst

```diff
@@ -305,6 +305,40 @@ Usage example::
     Simulation of a special unplanned case
     #noqa
 
+``checkendline``
+~~~~~~~~~~~~~~~~~
+
+Ensures that a blank line is kept at the end of each file.
+
+We want to keep diffs as clean as possible, and this allows us to append further changes to
+the end of a file without a messy diff.
+
+This is also important for some tools, so that reading from an input stream knows when to terminate.
+
+Usage example::
```

Comment:
If it can't be used like this, then why do we have a "Usage example"?

Reply:
Apologies, I think I left it there because at first I was looking for a command line command like the rest, I’ll clean this up